<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Waypoints</class>
 <widget class="QWidget" name="Waypoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pointAdd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePoints">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save points</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openPoints">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Open points</string>
   </property>
  </widget>
  <widget class="QListWidget" name="pointsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pointDelete">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete point</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
